test cases/common/234 very long commmand line/meson.build

```diff
@@ -0,0 +1,33 @@
+project('very long command lines', 'c')
+
+name = 'ALongFilenameMuchLongerThanIsNormallySeenAndReallyHardToReadThroughToTheEndAMooseOnceBitMySisterSheNowWorksAtLLamaFreshFarmsThisHasToBeSoLongThatWeExceed256KBWithoutCompilingTooManyFiles'
+namelen = 187
+if build_machine.system() == 'windows'
+  # ninja uses CreateProcess, and avoids cmd.exe
+  # CreateProcess has a limit of 32767; cygwin has a limit of 32000
```

Comment:
Hmmm... I don't think the "cygwin" comment belongs here (since CreateProcess isn't (directly) used by ninja, and system won't be 'windows' in that case)

Reply:
Fixed...